README.md

```diff
@@ -33,7 +33,7 @@
 
 # Quick Start
 
-Add following code to your readme.  
+Add the following code to your readme.  
 Change the `?username=` value to your GitHub's username.
```

Comment:
Perhaps these can be combined into one sentence.

Reply:
Updated so it is combined to one sentence.